lib/phoenix/digester.ex

```diff
@@ -68,17 +99,23 @@ defmodule Phoenix.Digester do
   end
 
   defp map_file(file_path, input_path) do
+    {:ok, stats} = File.stat(file_path)
     %{absolute_path: file_path,
       relative_path: Path.relative_to(file_path, input_path) |> Path.dirname(),
       filename: Path.basename(file_path),
+      mtime: stats.mtime,
```

Comment:
I believe rails just uses the current time for new files. It doesn't read mtime from the file itself. I vaguely remember that they used to do that and changed it for some reason. I could be wrong on that, but the safest thing may be to do what they do.